osgp/platform/osgp-core/src/main/resources/db/migration/V20201027172100000__Replaces_Event_deviceId_by_deviceIdentification.sql

```diff
@@ -0,0 +1,21 @@
+DO
+$$
+BEGIN
+
+  IF NOT EXISTS(SELECT 1 FROM information_schema.columns
+                WHERE table_schema = current_schema AND table_name = 'event' AND column_name = 'device_identification')
+  THEN
+    ALTER TABLE event ADD COLUMN device_identification CHARACTER VARYING(40),
```

Comment:
Shouldn't a NOT NULL constraint be added once the column is filled with data?

Reply:
The original "device" did not have one, therefore not added here, to be on the safe side.